.github/workflows/ci.yml

```
@@ -26,7 +26,7 @@ jobs:
 
       - run: npm ci
       - run: tsc --project tsconfig.production.json
-      - run: npx --no-install eslint app App.tsx
+      - run: npm run lint
```

```suggestion
      - run: npx --no-install eslint app App.tsx
```

need to revert this change because ci.yml must run without `--fix`